src/components/Sections/FeaturedSectionTextRight/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import './style.scss'
+import { Spacer } from '../../Spacer'
+
+export const FeaturedSectionTextRight = ({
+    listItem,
+    headerText,
+    descriptionText,
+    image,
+    color = 'yellow',
+    imgDesktopHeight = '413px',
+    imgDesktopWidth = '621px',
+    mobileSpacerHeight = '50px',
```

Comment:
These could be integers (`imgDesktopHeight = 413`) instead of strings with `px` added to them.